### DISCUSSION TITLE
registerBiometricSecret without inviting user to place his finger

### DISCUSSION THREAD

**Discussion by ghost** [NiklasMerz/cordova-plugin-fingerprint-aio#327] [Ideas]
Hello ,
First of all I wanted to thank you for this amazing plugin.
I was wondering if there is a way to avoid showing the screen where we invite the user to place his finger for authentication  during the registerBiometricSecret ? In case we want to update the secret ( when session has expired for example) we would like to avoid bothering the user from placing his finger. Is there a way to such a thing?
Thanks in advance,
Kristi.

**Comment by NiklasMerz** [comment]
To my knowledge this is required for the encryption part. But you could check the code to find details.
